framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -253,6 +253,8 @@
 
     /// Amount of detail in messages
 
+    // Print the version of the Checker Framework
+    "version",
```

Comment:
This user-visible change needs to be documented in the manual and in the changelog.